projects/angular/components/ui-suggest/src/ui-suggest.intl.ts

```diff
@@ -54,7 +54,9 @@ export class UiSuggestIntl implements OnDestroy {
      * @param itemNo The current item index.
      * @param itemCount The total item count.
      */
-    currentItemLabel = (text: string, itemNo: number, itemCount: number) => `${text} item ${itemNo} out of ${itemCount}`;
+    currentItemLabel = (text: string, itemNo: number, itemCount: number, selectedStatus?: boolean) => `${text}${this._selectedStatusText(selectedStatus)} item ${itemNo} out of ${itemCount}`;
```

Comment:
Since `_selectedStatusText` only does a boolean check why not make this a ternary? 
```suggestion
    currentItemLabel = (text: string, itemNo: number, itemCount: number, selectedStatus?: boolean) => selectedStatus 
    ? `${text} (selected) item ${itemNo} out of ${itemCount}`;
    : `${text} item ${itemNo} out of ${itemCount}`;
```